common-compose.yml

```diff
@@ -0,0 +1,70 @@
+wordpress:
```

Comment:
Ah, yes.

I've been using links lazily in my NFS example, but we've been using DNS to connect all the instances to Consul in most of our recent examples. Take a look at our ELK blueprint:

1 Give Consul a Triton CNS name in the Compose yaml: https://github.com/autopilotpattern/elk/blob/master/docker-compose.yml#L113-L114
1. Use an `_env` file to specify the DNS location; example: https://github.com/autopilotpattern/elk/blob/master/_env_example
1. Generate the `_env` for people in our `test.sh` script: https://github.com/autopilotpattern/elk/blob/master/test.sh#L168
1. Inject the `_env` in the Compose yaml: https://github.com/autopilotpattern/elk/blob/master/docker-compose.yml#L15 

Note that Triton CNS must be on. The `test.sh` script in ELK will confirm that, but here's where you turn it on: https://www.joyent.com/blog/introducing-triton-container-name-service#using-triton-cns-in-the-myjoyentcom-portal